package.json

```diff
@@ -9,7 +9,7 @@
     "deploy": "cross-env NODE_ENV=production webpack -p",
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack --config webpack.config.js",
-    "start": "webpack-dev-server --config webpack.config.js"
+    "start": "webpack serve --config webpack.config.js"
```

Comment:
While the command was changed here, if you don't install `webpack-dev-server` and try running `webpack serve` it will throw an error
> For using 'serve' command you need to install: 'webpack-dev-server' package.